VectorRumble.Shared/Helper.cs

```diff
@@ -35,7 +35,12 @@ public static string GetAssemblyTitle ()
 
         public static string[] GetFilesFromFolders (string [] folders, string extensionFilter)
         {
-            var directory = Path.Combine(folders);
+            string root = string.Empty;
```

Comment:
Will this change work locally too?

Reply:
It should do since 

```
var s = Path.Combine ("", "Foo");                                                                                                                                                        
```

`s` will be "Foo"